src/AppBundle/Entity/User.php

```diff
@@ -78,9 +78,9 @@ class User extends BaseUser
     /**
      * @var bool
      *
-     * @ORM\Column(type="boolean")
+     * @ORM\Column(type="boolean", nullable=true)
```

Comment:
@jongotlin How would you handle now pre-selected value for want_to_learn field? It doesnt mean that boolean values can't be nullable.